src/System.Reflection.Metadata/tests/Metadata/MetadataReaderTests.cs

```diff
@@ -2432,6 +2432,39 @@ public void Bug17109()
         }
 
         [Fact]
+        public void OtherAccessors()
+        {
+            var reader = GetMetadataReader(Interop.OtherAccessors);
+            var typeDef = reader.GetTypeDefinition(reader.TypeDefinitions.First());
+            Assert.Equal(reader.GetString(typeDef.Name), "<Module>");
+
+            typeDef = reader.GetTypeDefinition(reader.TypeDefinitions.Skip(1).First());
+            Assert.Equal(reader.GetString(typeDef.Name), "IContainerObject");
+
+            var propertyDef = reader.GetPropertyDefinition(typeDef.GetProperties().First());
+            var propertyAccessors = propertyDef.GetAccessors();
+
+            Assert.Equal("get_Value", reader.GetString(reader.GetMethodDefinition(propertyAccessors.Getter).Name));
+            Assert.Equal("set_Value", reader.GetString(reader.GetMethodDefinition(propertyAccessors.Setter).Name));
+            Assert.Equal("let_Value", reader.GetString(reader.GetMethodDefinition(propertyAccessors.Others.Single()).Name));
+
+            typeDef = reader.GetTypeDefinition(reader.TypeDefinitions.Skip(2).First());
+            Assert.Equal(reader.GetString(typeDef.Name), "IEventSource");
+
+            var eventDef = reader.GetEventDefinition(typeDef.GetEvents().First());
+            var eventAccessors = eventDef.GetAccessors();
+            var otherAccessorNames = (from methodHandle in eventAccessors.Others
+                                      select reader.GetString(reader.GetMethodDefinition(methodHandle).Name)).ToArray();
+
+            Assert.Equal("add_Notification", reader.GetString(reader.GetMethodDefinition(eventAccessors.Adder).Name));
+            Assert.Equal("remove_Notification", reader.GetString(reader.GetMethodDefinition(eventAccessors.Remover).Name));
+            Assert.True(eventAccessors.Raiser.IsNil);
+
+            // ilasm doesn't retain the order in which other accessors were specified in IL
+            AssertEx.SetEqual(otherAccessorNames, "suspend_Notification", "resume_Notification", "other_Notification");
```

Comment:
>AssertEx [](start = 12, length = 8)

Nit: even if ilasm doesn't retain order we have the dll checked in, so we can test the exact order. If the dll is regenerated then the test might need to be updated but that's ok.